28/07/2016 11:04:29 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:08:11 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:11:02 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:20:19 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
28/07/2016 11:20:19 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.basic.image.ZImageTest1.getResult(ZImageTest1.java:20)
	at br.zul.zwork2.test.basic.image.ZImageTest1.getResult(ZImageTest1.java:10)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 11:20:19 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 11:22:52 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:48:00 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:51:59 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:52:12 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:53:32 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:53:52 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 11:55:06 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:04:39 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:07:15 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:07:42 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:08:57 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:09:09 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:11:18 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:13:08 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
28/07/2016 12:13:08 - ERROR - java.lang.NullPointerException
	at javax.swing.ImageIcon.<init>(ImageIcon.java:205)
	at br.zul.zwork2.io.ZImage.fromResource(ZImage.java:46)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:19)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:10)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:13:08 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:13:27 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
28/07/2016 12:13:27 - ERROR - java.lang.NullPointerException
	at javax.swing.ImageIcon.<init>(ImageIcon.java:205)
	at br.zul.zwork2.io.ZImage.fromResource(ZImage.java:46)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:19)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:10)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:13:27 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:15:12 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
28/07/2016 12:15:12 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.io.ZImage.fromResource(ZImage.java:46)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:19)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:10)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:15:12 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:15:36 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:18:53 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
28/07/2016 12:18:53 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.io.ZImage.fromResource(ZImage.java:46)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:19)
	at br.zul.zwork2.test.basic.image.ZImageTest2.getResult(ZImageTest2.java:10)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:18:53 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
28/07/2016 12:19:35 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
28/07/2016 12:19:57 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
30/07/2016 14:46:20 - ERROR - java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:627)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:44)
	at br.zul.zwork2.http.ZHttpGet.send(ZHttpGet.java:28)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:23)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:12)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:532)
	... 12 more
30/07/2016 14:46:20 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
30/07/2016 14:46:20 - ERROR - java.lang.RuntimeException: java.net.MalformedURLException
	at br.zul.zwork2.log.ZPrintLog.prepareException(ZPrintLog.java:50)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:57)
	at br.zul.zwork2.http.ZHttpGet.send(ZHttpGet.java:28)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:23)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:12)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
Caused by: java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:627)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:44)
	... 9 more
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:532)
	... 12 more
30/07/2016 14:46:20 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
30/07/2016 14:47:08 - ERROR - br.zul.zwork2.http.ZHttpGet.prepareConnection():Erro em conectar com 'null'!
30/07/2016 14:47:08 - ERROR - java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:627)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:44)
	at br.zul.zwork2.http.ZHttpGet.send(ZHttpGet.java:28)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:23)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:12)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:532)
	... 12 more
30/07/2016 14:47:08 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
30/07/2016 14:47:08 - ERROR - java.lang.RuntimeException: java.net.MalformedURLException
	at br.zul.zwork2.log.ZPrintLog.prepareException(ZPrintLog.java:50)
	at br.zul.zwork2.log.ZPrintLog.prepareException(ZPrintLog.java:56)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:57)
	at br.zul.zwork2.http.ZHttpGet.send(ZHttpGet.java:28)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:23)
	at br.zul.zwork2.test.basic.http.ZHttpTest1.getResult(ZHttpTest1.java:12)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
Caused by: java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:627)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at br.zul.zwork2.http.ZHttpBase.prepareConnection(ZHttpBase.java:44)
	... 9 more
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:532)
	... 12 more
30/07/2016 14:47:08 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
14/08/2016 15:30:11 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 15:33:34 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 15:36:14 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 15:37:54 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 16:39:39 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 16:39:42 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 16:41:21 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 16:43:38 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
14/08/2016 16:44:10 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
22/09/2016 15:12:21 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
22/09/2016 15:12:21 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.key.ZKeyList.addKey(ZKeyList.java:39)
	at br.zul.zwork2.iterator.ZHtmlIterator.beforeFirst(ZHtmlIterator.java:221)
	at br.zul.zwork2.iterator.ZIterator.init(ZIterator.java:37)
	at br.zul.zwork2.iterator.ZIterator.<init>(ZIterator.java:30)
	at br.zul.zwork2.iterator.ZHtmlIterator.<init>(ZHtmlIterator.java:28)
	at br.zul.zwork2.iterator.ZHtmlIterator.<init>(ZHtmlIterator.java:36)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:34)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:13)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:12:21 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:12:52 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
22/09/2016 15:12:52 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.iterator.ZHtmlIterator.hasBefore(ZHtmlIterator.java:255)
	at br.zul.zwork2.iterator.ZHtmlIterator.next(ZHtmlIterator.java:172)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:36)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:13)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:12:52 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:15:09 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
22/09/2016 15:15:29 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
22/09/2016 15:15:40 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
22/09/2016 15:19:06 - ERROR - br.zul.zwork2.test.ZTestManager.executeTest(ZCustomTest test):ERRO!
22/09/2016 15:19:06 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.iterator.ZHtmlIterator.hasBefore(ZHtmlIterator.java:267)
	at br.zul.zwork2.iterator.ZHtmlIterator.next(ZHtmlIterator.java:172)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:36)
	at br.zul.zwork2.test.basic.html.ZHtmlTest2.getResult(ZHtmlTest2.java:13)
	at br.zul.zwork2.test.ZTest.run(ZTest.java:18)
	at br.zul.zwork2.test.ZMultipleTests.run(ZMultipleTests.java:29)
	at br.zul.zwork2.test.ZTestManager.executeTest(ZTestManager.java:123)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:65)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:19:06 - ERROR - java.lang.NullPointerException
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:68)
	at br.zul.zwork2.test.ZTestManager.executeTests(ZTestManager.java:47)
	at br.zul.zwork2.main.Main.main(Main.java:27)
22/09/2016 15:23:05 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
23/09/2016 09:18:48 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
27/09/2016 09:44:24 - ERROR - br.zul.zwork2.main.Main.main(String[] args):Quantidade de testes que falharam: 1
